lib/valkyrie/persistence/postgres/metadata_adapter.rb

```diff
@@ -2,6 +2,12 @@
 require 'valkyrie/persistence/postgres/persister'
 require 'valkyrie/persistence/postgres/query_service'
 module Valkyrie::Persistence::Postgres
+  # Metadata Adapter for Postgres Adapter.
```

Comment:
@tpendragon I have been referring to Postgres, Solr ect as a backend.  Is that correct, or is Adapter more correct?